src/drake/shell.clj

```diff
@@ -96,7 +102,7 @@
         proc (.exec (Runtime/getRuntime)
```

Comment:
Not sure what require has to do with this. Can't you just test a clojure program that writes to ./foo.txt and see where the file ends up?